src/game.cpp

```diff
@@ -4622,6 +4631,9 @@ void Game::internalDecayItem(Item* item)
 		Item* newItem = transformItem(item, item->getDecayTo());
 		startDecay(newItem);
 	} else {
+		if (Player* player = item->getHoldingPlayer()) {
+			player->sendSupplyUsed(it.transformDeEquipTo != 0 ? Item::items[it.transformDeEquipTo].clientId : item->getClientID());
```

Comment:
That's weird, are there any items that decay and they are supply?

Reply:
rings

Reply:
> That's weird, are there any items that decay and they are supply?

rings